server/flaskr/database/postgres/handlers/user_data_handler.py

```
@@ -9,24 +11,47 @@ def create_user(cls, firstName: str, lastName: str, email: str, passwordHash: st
             with get_db_access() as conn:
                 cur = conn.cursor()
 
-                query = f"INSERT INTO users (firstName, lastName, passwordHash, email) values (%s, %s, %s, %s) RETURNING id;"
+                query = f"INSERT INTO public.users (firstName, lastName, passwordHash, email) values (%s, %s, %s, %s) RETURNING id;"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)